package.json

```
@@ -5,9 +5,12 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint --max-warnings 0",
-    "lint:fix": "next lint --max-warnings 0 --fix",
-    "format": "prettier --write --loglevel=warn '*/**/*.{js,jsx,ts,tsx,html,gql,graphql,json}'",
+    "lint": "yarn lint:eslint & yarn lint:prettier",
+    "lint:eslint": "eslint --ext .ts,.tsx,.js .",
+    "lint:prettier": "prettier --ignore-path .prettierignore  --check '**/*.{js,ts,tsx,css}'",
+    "format": "yarn format:eslint & yarn format:prettier",
```

[imo]
設定によりますが気になったのでコメントだけ。
`&` で実行させる方が非同期で動くので早いのですが、実行タイミングによっては差分が出てしまう場合がありそうだなと思いました。
遅くはなりますが同期して動かすことで、常に一定のformat結果が得られた方が良いかなと思いました。

```suggestion
    "format": "yarn format:eslint && yarn format:prettier",
```